### ISSUE TITLE
"Disable all features" for sites ignored

### ISSUE THREAD

**Issue by sgrossberndt** [keepassxreboot/keepassxc-browser#545]
## Expected Behavior

When "Disable all features" `[ignoreFull]` has been set for a site (e.g. "https://twitter.com/") in the site preferences KeepassXC-Browser should not be active when visiting that site. 

- When "Disable all features" is set for a site scanning of input elements on that site must be completely disabled
- `A script of KeepassXC-Browser is causing Firefox to slow down` should never show up

## Current Behavior

Although "Disable all features" has been set for a site (e.g. "https://twitter.com/") in the site preferences when visiting that site KeepassXC-Browser is still active.
I noticed this as on subpages in an TYPO3 backend with lots of `<input>` elements I get the message `A script of KeepassXC-Browser is causing Firefox to slow down` (maybe related to https://github.com/keepassxreboot/keepassxc-browser/issues/146 ?). As this makes the browser so slow its unusable I have to disable and enable the addon all the time :-(

## Steps to Reproduce (for bugs)
1. Add 'https://twitter.com/' to the list of sites in `site preferences`
2. Select "Disable all features" for that site.
3. Visit "https://twitter.com/"
4. Notice the KeepassXC icon is still green (and causing slowdowns on pages of that site containing many input elements)

## Debug info
KeePassXC - 2.4.1
KeePassXC-Browser - 1.4.3
Operating system: Win
Browser: Firefox
```
Error: Load timeout for modules: TYPO3/CMS/Backend/ContextMenuActions
http://requirejs.org/docs/errors.html#timeout require.js:1:1067
    makeError http://sevas.lan/typo3/sysext/core/Resources/Public/JavaScript/Contrib/require.js:1
    n http://sevas.lan/typo3/sysext/core/Resources/Public/JavaScript/Contrib/require.js:1
    completeLoad http://sevas.lan/typo3/sysext/core/Resources/Public/JavaScript/Contrib/require.js:1
    onScriptLoad http://sevas.lan/typo3/sysext/core/Resources/Public/JavaScript/Contrib/require.js:1
```

**Comment by varjolintu** [comment]
Use `https://twitter.com/*` as the URL.

**Comment by sgrossberndt** [comment]
Does not make a difference.

**Comment by varjolintu** [comment]
I cannot reproduce this. Could you make some performance tests using Web Developer? Also, maximum number of input fields are restricted to 100 per page mostly because of the possible slowdowns. When I tested Twitter and surfed around it, the count of input fields stays quite small.

**Comment by sgrossberndt** [comment]
[web-developer-profiles.json.zip](https://github.com/keepassxreboot/keepassxc-browser/files/3183564/web-developer-profiles.json.zip)
Attached are two profile recordings from firefox web developer tools of loading a page in a TYPO3 backend containing many input elements - one with keepassxc-browser addon enabled and this site added as "http://example.dev/*" with "Disable all features" and the second with the keepassxc-browser addon disabled completely. Difference: 10 seconds vs 800 ms.

I chose "https://twitter.com/" as an example to show "Disable all features" for sites ignored as KeepassXC-Browser is still active, I did not notice any slowdowns on twitter.com - those are a issue on their own for pages with many input eIements I guess.
